lib/modules/@apostrophecms/modals/src/apos/components/ApostropheModal.vue

```diff
@@ -37,15 +37,15 @@ export default {
 
```

Comment:
Will this file be getting renamed AposModal for consistency with the newer components?

Reply:
That one will be deleted. I didn't mean to request the new review. My bad.

Reply:
I'm going to leave this in for now to get this merged in for the sake of being able to have others break off pieces to work on. We'll be back for this and the editor modal.